src/lib.rs

```diff
@@ -116,6 +116,7 @@ mod token;
 pub use event::{
     PollOpt,
     EventSet,
+    IoEvent,
```

Comment:
How come this ended up being exported? Shouldn't this be an internal implementation detail?